custom_components/tuya_local/devices/pv28-cw_airquality_monitor.yaml

```diff
@@ -0,0 +1,299 @@
+  - id: yrr3eiyiacm31ski
+    name: PV28-CW
+primary_entity:
+  entity: sensor
+  name: CO2
+  class: carbon_dioxide
+  dps:
+    - id: 2
+      type: integer
+      name: sensor
+      unit: ppm
+      class: measurement
+
+secondary_entities:
+  - entity: binary_sensor
+    name: CO2 levels
+    class: safety
+    dps:
+      - id: 1
+        type: string
+        name: sensor
+        mapping:
+          - dps_val: "alarm"
+            value: true
+          - dps_val: "normal"
+            value: false
+  # - entity: select
+  #   name: Alarm Volume
+  #   category: config
+  #   dps:
+  #     - id: 5
+  #       type: string
+  #       name: option
+  #       optional: true
+  #       mapping:
+  #         - dps_val: low
+  #           value: "low"
+  #         - dps_val: middle
+  #           value: "middle"
+  #         - dps_val: high
+  #           value: "high"
+  #         - dps_val: mute
+  #           value: "mute"
+  # - entity: select
+  #   name: Alarm Melody
+  #   category: config
+  #   dps:
+  #     - id: 6
+  #       type: string
+  #       name: option
+  #       optional: true
+  #       mapping:
+  #         - dps_val: "ringtone_1"
+  #           value: "ringtone_1"
+  #         - dps_val: "ringtone_2"
+  #           value: "ringtone_2"
+  #         - dps_val: "ringtone_3"
+  #           value: "ringtone_3"
+  #         - dps_val: "ringtone_4"
+  #           value: "ringtone_4"
+  #         - dps_val: "ringtone_5"
+  #           value: "ringtone_5"
+  # - entity: number
+  #   name: Alarm Duration
+  #   category: config
+  #   dps:
+  #     - id: 7
+  #       type: integer
+  #       name: value
+  #       optional: true
+  #       unit: s
+  #       range:
+  #         min: 1
+  #         max: 60
+  #       mapping:
+  #         - step: 1
+  - entity: switch
+    name: Alarm Switch
+    category: config
+    dps:
+      - id: 13
+        type: boolean
+        name: switch
+        mapping:
+          - dps_val: true
+            icon: "mdi:bell"
+          - dps_val: false
+            icon: "mdi:bell-off"
+  - entity: sensor
+    name: Power State
+    class: enum
+    category: diagnostic
+    dps:
+      - id: 14
+        type: string
+        name: sensor
+        mapping:
+          - dps_val: normal
+            value: battery
+          - dps_val: charge
+            value: charging
+  - entity: sensor
+    name: Battery
+    class: battery
+    category: diagnostic
+    dps:
+      - id: 15
+        type: integer
+        name: sensor
+        unit: "%"
+        class: measurement
+  # - entity: number
+  #   name: Screen Brightness
+  #   category: config
+  #   dps:
+  #     - id: 17
+  #       type: integer
+  #       name: value
+  #       unit: "%"
+  #       range:
+  #         min: 0
+  #         max: 100
+  #       mapping:
+  #         - step: 1
+  - entity: sensor
+    name: Temperature
+    class: temperature
+    dps:
+      - id: 18
+        type: integer
+        name: sensor
+        unit: "°C"
```

Comment:
I'm not sure if Home Assistant is expecting "U+00B0 C" here or "U+2103".  It might be safer to use just C, which has a special case translation to TEMP_CELSIUS in the code.

Reply:
changed to C

Reply:
You don't seem to have committed this change